inst/WORDLIST

```diff
@@ -1,3 +1,4 @@
+ADaM
 CDISC
```

Comment:
:+1: I should've looked in other packages

Glancing at `{teal.slices}` it actually looks good on the pkgdown site. It may be just local vignettes that look weird.